recipes/formulaic-contrasts/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "formulaic-contrasts" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/formulaic_contrasts-{{ version }}.tar.gz
+  sha256: 0a575a810bf1fba28938259d86a3ae2ae90cb9826fca84b9409085170862f701
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('formulaic-contrasts', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - hatchling
```

I think we also need [`hatch-vcs`](https://github.com/scverse/formulaic-contrasts/blob/1b063d86bc72aeb8034917f7a637d17a4c50a768/pyproject.toml#L3)
```suggestion
    - hatchling
    - hatch-vcs
```